ui/src/app/views/homepage/component.html

```diff
@@ -1,80 +1,135 @@
+                                    Workspaces</a>
+                            </div>
+                            <div class="col">
+                                <a class="links disabled">See All Cohorts</a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="card add-card" (click)="addWorkspace()">
+                        <div class="card-text">
+                            Create a <br/> New Workspace <br/>
+                            <clr-icon shape="plus-circle" class="plus-circle"></clr-icon>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+        </div>
     </div>
-<div class="card-area">
-    <div class="card">
-        <div class="title">Create your own workspace</div>
-        <div class="description">Here you can create and name a workspace.
-            Within a workspace you can create notebooks and build cohorts.</div>
-        <button class="button btn btn-primary tooltip" (click)="addWorkspace()"
-            [disabled]="!billingProjectInitialized || !hasCdrAccess()">
-          CREATE WORKSPACE
-          <span *ngIf="!billingProjectInitialized" class="tooltip-content">
-              Your billing project is currently being created, please wait.
-          </span>
-          <span *ngIf="billingProjectInitialized && !hasCdrAccess()" class="tooltip-content">
-              You must complete profile setup before accessing data.
-          </span>
-        </button>
-    </div>
-    <div class="card">
-        <div class="title">View your workspaces</div>
-        <div class="description">Here you can view all workspaces you have created, or that have been
-            shared with you.</div>
-        <button class="button btn btn-primary tooltip" (click)="listWorkspaces()"
-            [disabled]="!hasCdrAccess()">
-          GO TO WORKSPACES
-          <span *ngIf="!profile" class="tooltip-content">
-              Your account is being initialized, please wait.
-          </span>
-          <span *ngIf="profile && !hasCdrAccess()" class="tooltip-content">
-              You must complete profile setup before accessing data.
-          </span>
-        </button>
-    </div>
-</div>
-<div class="bottom-banner">
-    <div class="logo">
-        <img [src]="AouLogoFooter">
-    </div>
-    <div class="bottom-links">
-        Privacy Policy
-    </div>
-    <div class="bottom-links">
-        Terms of Service
+    <div class="footer">
+        <div class="row">
+            <div class="footerTitle">How to Use the AoU Researcher Workbench</div>
+        </div>
+        <div class="row textGroup">
+            <div class="footerText">
+                <div class="title">Working Within Researcher Workbench</div>
+                <ul class="bullet">
+                    <li>Researcher Workbench Mission</li>
```

Comment:
Similarly, I would color these grey, and make them disabled. Currently they look like links, and I think that is not what we want.